File: src/main/angular/app/model/inserat-uebersicht.ts
import {InseratStatus} from "./inserat-status";

export interface InseratUebersicht {
    id: number;
    lastUpdate: Date;
    rufname: string;
    status: InseratStatus;
    alter: string;
    plz: string;
    ort: string;

    aktivierbar:boolean;
    deaktivierbar:boolean;
    vermittelbar:boolean;
    loeschbar:boolean;
    editierbar:boolean;

    vermittler:string;
    storyschreiber: string;
}

File: src/main/java/com/github/dmn1k/tfm/inserate/InseratUebersicht.java
package com.github.dmn1k.tfm.inserate;

import lombok.*;


@Builder
@Value
public class InseratUebersicht {
    private Long id;
    private String lastUpdate;
    private String rufname;
    private String rasse;
    private String alter;
    private String plz;
    private String ort;
    private InseratStatus status;
    private String vermittler;

    private boolean aktivierbar;
    private boolean deaktivierbar;
    private boolean vermittelbar;
    private boolean loeschbar;
    private boolean editierbar;
}
